src/components/widgets/ItemMenu.js

```diff
@@ -108,16 +109,16 @@ const ItemMenu = props => {
   };
 
   return (
-    <div style={{ background: '#fff' }}>
+    <Paper>
       <Row type="flex" justify="start">
-        <Col xs={24} sm={24} md={7} lg={5}>
+        <Col style={{ width: width <= SMALL ? '100%' : '200px' }}>
           {renderMenu()}
         </Col>
-        <Col xs={24} sm={24} md={17} lg={19} style={{ padding: '16px', boxSizing: 'border-box' }}>
+        <Col style={{ width: width <= SMALL ? '100%' : 'calc(100% - 200px)', padding: '16px', boxSizing: 'border-box' }}>
```

Comment:
Looks like this has changed more than intended? Might be a different commit. The vertical line is now not full height.
![screen shot 2019-01-07 at 10 05 03](https://user-images.githubusercontent.com/6759267/50760912-5cd77180-1269-11e9-8513-c8ea878bb39b.png)

I'm wondering if a nicer way to do this would be to simply add a limit to the card width: maxWidth:1200, margin:auto;?